packages/bigtable/system-test/bigtable.js

```diff
@@ -510,7 +509,7 @@ describe('Bigtable', function() {
       it('should get rows', function(done) {
         TABLE.getRows(function(err, rows) {
           assert.ifError(err);
-          assert.strictEqual(rows.length, 4);
+          assert(rows.length >= ROWS.length);
```

Comment:
Why >= from strict equal? Could the rows returned be more than the rows provided?